src/main/kotlin/org/rust/ide/settings/RsCodeCompletionOptions.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.settings
+
+import com.intellij.application.options.CodeCompletionOptionsCustomSection
+import com.intellij.ui.IdeBorderFactory
+import com.intellij.ui.components.JBCheckBox
+import com.intellij.ui.layout.panel
+import org.rust.openapiext.CheckboxDelegate
+import javax.swing.JPanel
+
+class RsCodeCompletionConfigurable : CodeCompletionOptionsCustomSection {
```

Comment:
Looks like `CodeCompletionOptionsCustomSection` is available since `183.4886 (IDEA 2018.3.2)`
Let's set `sinceBuild` to `183.4886` in `gradle-183.properties` and increase oldest IDEA version from `2018.3` to `2018.3.2` in `.travis.yml`

Reply:
Mmm, can we just drop support of these versions?

Reply:
Yep. The following actions 
> Let's set `sinceBuild` to `183.4886` in `gradle-183.properties` and increase oldest IDEA version from `2018.3` to `2018.3.2` in `.travis.yml`

just disable the plugin for all IDE's below `2018.3.2`